client/app/templates/modals/editEntry.jade

```diff
@@ -1,4 +1,4 @@
 div.entry-modal
  textarea(ng-model="currentEntry.text")
  button.button.showy(ng-click='$dismiss(this)') Cancel
- button.button.showy(ng-click='$close(this)') OK
+ button.button.showy.disabled(ng-click='$close(this)' ng-disabled="!currentEntry.text") OK
```

Comment:
I think that isn't necessary to add the ```.disabled``` class since the directive ```ng-disabled``` does that dynamically based on the evaluation